frontend/lang/en.json

```diff
@@ -221,7 +223,7 @@
   "runningSession.checkboxes.unpublished": "Unpublished",
   "runningSession.confusion.difficulty.Range": "easy - hard",
   "runningSession.confusion.difficulty.Title": "Difficulty",
-  "runningSession.confusion.info": "The Confusion-Barometer allows you to get feedback on the speed and difficulty of your lecture as it evolves over time.",
+  "runningSession.confusion.info": "The Confusion-Barometer allows you to get feedback on the speed and difficulty of your lecture as it evolves over time. It displays all feedbacks that were shared by students throughout the past 10 minutes",
```

Comment:
when I run `npm run build` on my machine, it extracts all of the language strings and builds this `en.json` again. this string here is removed entirely, as it is not used anymore in the code. if we quickly run `npm run build` when a branch is ready to merge, we can ensure that no old strings are kept around in the code. to remove them from `de.json` as well, we would need to remove them by hand and run `npm run build` again, so they are also removed from the compiled version of `de`

for this string here, we could maybe keep it around and put it inside a tooltip. maybe you could investigate adding a tooltip in the confusion barometer view

Reply:
The tooltip is a great idea, will definitely look into it - so we can omit all disruptive text in this view

Reply:
tooltip was added